### ISSUE TITLE
New coins zergpool. The heck frkhash?

### ISSUE THREAD

**Issue by Robbylikes** [RainbowMiner/RainbowMiner#2626]

**Comment by Robbylikes** [comment]
I see people mining it but rainbow won't.

**Comment by RainbowMiner** [comment]
SrbminerMulti should support FrkHash - I will check, why it is not showing up.

**Comment by RainbowMiner** [comment]
> SrbminerMulti should support FrkHash - I will check, why it is not showing up.

No, not yet. We will have to integrate FrkHash miner, but it's Nvidia and CPU, only.

**Comment by Robbylikes** [comment]
All I have in Nvidia so please add away.

**Comment by RainbowMiner** [comment]
New ZergPool coins / frkhash: ZergPool was removed from RainbowMiner on 2025-09-27 (b4f6f448). Closing as obsolete for the current master (v5.0.2.1) - please open a new issue if this still occurs. Thanks for the report!
